_pages/faq.md

```diff
@@ -8,123 +8,185 @@ accordion-quick-links: true
-    - group-heading: "I am a company or institution"
-    - group-heading: "Contact MLab"
+    - group-heading: "About M-Lab"
+    - group-heading: "Supporting or Contributing to M-Lab"
 gq-accordion:
-  - heading: "What and who are M-lab's supporting partners?"
-  - heading: "What's M-Lab's origin story?"
-  - heading: "Who's behind M-Lab? Can I get involved?"
-  - heading: "What is M-Lab, and why?"
-iu-accordion:
-  - heading: "How can I troubleshoot the NDT speed test?"
-  - heading: "A test isn't working. Whom do I contact?"
+  - heading: "What is M-Lab?"
+td-accordion:
+  - heading: "What measurement tests can I run?"
+  - heading: "Will these tests monitor my private Internet activity, like email messages or, browsing and search history?"
   - heading: "What data will be collected when I run a test?"
   - heading: "Will these tests monitor my private Internet activity, like email, browsing, search?"
-  - heading: "Are the measurement results always right?"
-  - heading: "What measurement tests can I run?"
-researcher-accordion:
-  - heading: "Can I deploy a new tool on the M-Lab platform?"
-company-accordion:
-  - heading: "How can I get involved?"
-contact-accordion:
+  - heading: "What data will be collected when I run a test?"
+  - heading: "Who is collecting data? Where is it stored?"
+  - heading: "Where can I explore M-Lab's data?"
+  - heading: "Why are my M-Lab results different from other speed tests?"
+  - heading: "How to Report Issues with M-Lab tests"
+ab-accordion:
+  - heading: "What is M-Lab's Mission?"
+  - heading: "Who are M-Lab's supporting partners?"
+  - heading: "What is the history of M-Lab?"
+  - heading: "How does M-Lab choose the tests it hosts?"
   - heading: "How can I contact M-Lab?"
+sc-accordion:
+  - heading: "How can I get involved and support M-Lab?"
+  - heading: "How can companies or other organizations get involved and support M-Lab?"
+  - heading: "How can researchers get involved?"
+  - heading: "Can I use M-Lab's platform to host my own measurement experiment?"
+  - heading: "How can I add an M-Lab test to my website, app or other software?"
 ---
 
 {:.general-questions}
 # General questions
 
 {% capture accordion_entry_1 %}
-Supporting partners are companies and institutions that have committed resources to M-Lab, either in the form of server infrastructure, funding, data storage/analysis, staffing, or other services dedicated to furthering M-Lab's goals. Check out our [list of supporting partners]({{ site.baseurl }}/who/).
-{% endcapture %}
+Measurement Lab (M-Lab) is the largest open source Internet measurement effort in the world. M-Lab provides performance [tests]({{ site.baseurl }}/tests) for consumers to have an accurate picture of their Internet service, by offering a robust, state-of-the-art server platform that supports diverse measurement needs. The data is aggregated and released in the public domain for policymakers, researchers, and anyone interested in Internet issues.
```

Comment:
@critzo It looks like when this new content was inserted, the `{% endcapture %}` tag was removed, so Jekyll is choking on that I think.  Basically the content within the 2 capture tags will be what is in each accordion.